docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md

```diff
@@ -0,0 +1,24 @@
+# KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert
+
+## Meaning
+The `KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert` alerts means that the HypeConverged cluster operator (HCO) is running, but the `HyperConverged` custom resource does not exist.
+
+There are two reasons for that to happen: First, after installation of HCO, it is required to create the `HyperConverged` custom resource in order to complete the installation. The purpose of this alert is to guide the user to complete the installation by creating the `HyperConverged` custom resource. 
+
+The other reason for this alert is during uninstallation of HCO. In this case, the user should remove the `HyperConverged` CR before uninstalling the operator, and then again, the HCO is still running, while the `HyperConverged` custom resource was already removed, and so the alert will be triggered. In this case, the alert can be safely ignored. Completing the uninstallation process will cancel the alert.
+
+## Mitigation
+In case of installation, just create the HyperConverged custom resource to complete the installation; for example, to create the HyperConverged custom resource with default values, do:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: hco-operatorgroup
+  namespace: kubevirt-hyperconverged
+spec: {}
+EOF
+```
+
+In case of uninstallation, just ignore the alert and continue the process. After the HCO will stop running, the alert will be canceled.
```

Comment:
@nunnatsa

Reply:
@nunnatsa I think that if we check the condition for an hour the second option on uninstallation would be different. Perhaps the uninstallation is stuck no?